### ISSUE TITLE
EMQX Dashboard of limiter page occurred API_NOT_EXIST error

### ISSUE THREAD

**Issue by wangfeiing** [emqx/emqx#10982]
### What happened?

I use limiter with dashborad,404 error occurred
```
404 API_NOT_EXIST:Request Path Not Found
```

![image](https://github.com/emqx/emqx/assets/13382088/ca9bc2a8-d356-44ff-bb3a-6b2e6fac680d)

### What did you expect to happen?

dashboard should show page of limiter info

### How can we reproduce it (as minimally and precisely as possible)?

emqx version   : 5.0.23

### EMQX version

<details>

```console
$ ./bin/emqx_ctl broker
sysdescr  : EMQX
version   : 5.0.23
datetime  : 2023-06-08T12:36:33.800117148+00:00
uptime    : 4 hours, 36 minutes, 47 seconds
```

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
NAME="Tencent Linux"
VERSION="2.4"
ID="tlinux"
ID_LIKE="rhel fedora centos"
VERSION_ID="2.4"
PRETTY_NAME="Tencent Linux 2.4"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:tlinux:linux:2"
HOME_URL="https://tlinux.qq.com/"
$ uname -a
Linux 4.14.105-19-0022 #1 SMP Wed Sep 29 11:36:29 CST 2021 x86_64 x86_64 x86_64 GNU/Linux
```

</details>

### Log files

<details>

</details>

**Comment by ChuckSayHi** [comment]
It was deleted by a mistake, see: https://github.com/emqx/emqx/pull/10431
And now it has been fixed by this PR: [#10495](https://github.com/emqx/emqx/pull/10495).
This issue has been fixed in the new version: - [#10495](https://github.com/emqx/emqx/pull/10495) Add the limiter API `/configs/limiter` which was deleted by mistake back.

**Comment by zmstone** [comment]
Closing this issue as it's already fixed in v5.0.26
